src/main/java/beam/analysis/PathTraversalEventGenerationFromCsv.java

```diff
@@ -18,14 +18,11 @@ public static void main(String[] args) {
     }
 
     public static void generatePathTraversalEventsAndForwardToHandler(String filePath, PathTraversalSpatialTemporalTableGenerator handler) {
-        String csvFile = filePath;
-        BufferedReader br = null;
         String line = "";
 
-        try {
-            br = new BufferedReader(new FileReader(csvFile));
-            line = br.readLine(); // skipping header line
+        try(BufferedReader br = new BufferedReader(new FileReader(filePath))) {
```

Comment:
Very nice. I hadn't realized that Java had this. I liked it when in my .NET days - good to know it's available in Java :)